main.js

```diff
@@ -263,27 +264,47 @@ class ModuleInstance extends InstanceBase {
 					true
 				)
 			} else if ((matches = message.address.match(cmd))) {
-				let user_id = matches[1]
-				if (user_id === this.config.user_id || this.config.user_id === '-1') {
+				let userid = matches[1]
+				if (userid == this.config.userid || this.config.userid == '-1') {
 					this.setInstanceStates(
 						{
 							cmd: message.args[0].value,
 						},
 						true
 					)
 				}
-			}
-
-			// set variables/state for wheel values
-			else if ((matches = message.address.match(enc_wheel))) {
+			} else if (matches = message.address.match(chan)) {
+				// Thismay be a better place to reset our parameter data variables
+				let chantext
+				chantext = message.args[0].value
+				let chanarg_matches
+				chanarg_matches = chantext.match( /^(\d+)/ )
+				if ( chanarg_matches.length > 0 ) {
+					let actChan
+					actChan = chanarg_matches[1]
```

Comment:
Its a bit cleaner to set the value in the same line as defining the variable, there isnt a reason not to do that here
```suggestion
				let chantext = message.args[0].value
				let chanarg_matches = chantext.match( /^(\d+)/ )
				if ( chanarg_matches.length > 0 ) {
					let actChan = chanarg_matches[1]
```